cadence/src/sinks/mod.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-mod core;
+pub mod core;
```

Comment:
Instead of making `core` public, can you export the relevant structs below? We can still keep them confined to `ext` by not re-exporting them in `lib.rs`

Reply:
Fixed.